web/src/components/metrics.tsx

```diff
@@ -459,23 +472,30 @@ const QueryKebab: React.FC<{ index: number }> = ({ index }) => {
     dispatch(queryBrowserDuplicateQuery(index));
   }, [dispatch, index]);
 
-  const isSpan = (item) => item?.title?.props?.children;
+  type Item = {
+    title?: {
```

Comment:
we should avoid using any, I think in this case we know that title a ReactElement, which will include all other needed properties.